server/models/Feedback.js

```diff
@@ -1,39 +1,156 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
-const User = mongoose.model('User');
+const Interview = mongoose.model('Interview');
 
-const opts = {
-  // Make Mongoose use Unix time (seconds since Jan 1, 1970)
-  timestamps: { currentTime: () => Math.floor(Date.now() / 1000) },
-};
-
-const FeedbackSchema = new Schema(
-  {
-    createdAt: Number,
-    updatedAt: Number,
-    content: {
-      type: String,
+const FeedbackSchema = new Schema({
+  createdAt: Number,
+  updatedAt: Number,
+  performanceLevel: {
+    type: Number,
+  },
+  categories: {
+    communication: {
+      type: Number,
     },
-    codingRating: {
+    codeEfficiency: {
       type: Number,
     },
-    communication: {
+    codeOrganization: {
       type: Number,
     },
-    user: {
-      type: Schema.Types.ObjectId,
-      ref: 'User',
+    speed: {
+      type: Number,
     },
-    interview: {
-      type: Schema.Types.ObjectId,
-      ref: 'Interview',
+    debugging: {
+      type: Number,
+    },
+    prolemSolving: {
+      type: Number,
     },
   },
-  opts
-);
+  strengths: {
+    type: String,
+  },
+  improvements: {
+    type: String,
+  },
+  resources: {
+    type: String,
+  },
+  other: {
+    type: String,
+  },
+  feedbackCreator: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  feedbackReciever: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  interview: {
+    type: Schema.Types.ObjectId,
+    ref: 'Interview',
+  },
+});
+
+const Feedback = mongoose.model('Feedback', FeedbackSchema);
+
+async function addFeedback(req) {
+  const {
+    performanceLevel,
+    categories,
+    strengths,
+    improvements,
+    resources,
+    other,
+    userIdReciever,
+    interviewId,
+  } = req.body;
+
+  const userIdCreator = req.user._id;
+
+  try {
+    const interview = await Interview.findOne({ _id: interviewId });
+    let currFeedback = '';
+
+    let interviewUsers = interview.users[0];
+    if (!interview.users[0].user.equals(userIdCreator)) {
+      interviewUsers = interview.users[1];
+    }
```

Comment:
This kind of only works for just two users, I would instead just use a find (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)

Reply:
since there's only two users in an interview I thought we could just use the index

Reply:
Because it would be both more concise and more extensible in the future, I think we should use find as well rather than relying on the index